expression/constant.go

```diff
@@ -284,7 +412,6 @@ func (c *Constant) EvalJSON(ctx sessionctx.Context, _ chunk.Row) (json.BinaryJSO
 		if err != nil {
 			return json.BinaryJSON{}, true, err
 		}
-		fmt.Println("const eval json", val.GetMysqlJSON().String())
```

Comment:
???

Reply:
Could you use another PR to remove this line? We need to cherry pick this change to the release branches...